src/Core/ProtocolDefines.h

```diff
@@ -112,12 +112,14 @@ static constexpr auto DBMS_MIN_REVISON_WITH_PARALLEL_BLOCK_MARSHALLING = 54478;
 
 static constexpr auto DBMS_MIN_REVISION_WITH_VERSIONED_CLUSTER_FUNCTION_PROTOCOL = 54479;
 
+static constexpr auto DBMS_MIN_REVISION_WITH_COMPRESSED_LOGS_PROFILE_EVENTS_COLUMNS = 54480;
```

Comment:
We may also want to add a test for this, i.e. expose revision as a client argument, run with older revision, run with newer and check that at least we have less bytes send/receive over network.